test/mock-debug-test.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference path='../typings/chai/chai.d.ts'/>
+/// <reference path="../typings/mocha/mocha.d.ts"/>
+
+'use strict';
+
+import chai = require('chai');
+import { makeMockLogger, DebugLogger } from '../mock-debug';
```

Comment:
Perhaps. But I doubt anyone would actually use it as a drop-in replacement. I envision an API where one could pass in a logging function, not a debug module. I don't see an API where passing in the a debug module would be helpful. The test harness needs to instantiate a specific mock logging function, so that it can later call .get() on that specific function. If the API under test instead expected the debug module to be passed in, and it therefore had to create the debug logging function locally, the test harness would not have access to the logging function that held the history.